src/main/java/com/rultor/agents/github/ClosePullRequest.java

```diff
@@ -0,0 +1,103 @@
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the rultor.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents.github;
+
+import com.jcabi.aspects.Immutable;
+import com.jcabi.github.Github;
+import com.jcabi.github.Issue;
+import com.jcabi.xml.XML;
+import com.rultor.agents.AbstractAgent;
+import com.rultor.spi.Profile;
+import java.io.IOException;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.xembly.Directive;
+import org.xembly.Directives;
+
+/**
+ * Closes pull request manually, leaves comment with description.
+ *
+ * @author Viktor Kuchyn (kuchin.victor@gmail.com)
+ * @version $Id$
+ * @since 2.0
+ */
+@Immutable
+@ToString
+@EqualsAndHashCode(callSuper = false, of = {"github", "profile"})
+public final class ClosePullRequest extends AbstractAgent {
+
+    /**
+     * Profile.
+     */
+    private final transient Profile.Defaults profile;
+
+    /**
+     * Github.
+     */
+    private final transient Github github;
+
+    /**
+     * Constructor.
+     * @param prof Profile
+     * @param ghub Github
+     */
+    public ClosePullRequest(final Profile prof, final Github ghub) {
+        super(
+            "/talk/wire[github-repo and github-issue]"
+        );
+        this.profile = new Profile.Defaults(prof);
+        this.github = ghub;
+    }
+
+    @Override
+    public Iterable<Directive> process(final XML xml) throws IOException {
+        if (ClosePullRequest.isRebaseMode(this.profile)) {
+            final Issue.Smart issue = new TalkIssues(this.github, xml).get();
+            issue.close();
+            issue.comments().post("Closed manually because of rebase mode");
+        }
+        return new Directives();
+    }
+
+    /**
+     * Check rebase mode.
+     * @param prof Profile
+     * @return True if rebase mode, false - otherwise
+     * @throws IOException If errors
+     */
+    private static boolean isRebaseMode(final Profile.Defaults prof)
```

Comment:
@vkuchyn no need to pull out this code from "process()" method. It's not too big, not repetitive, not adding any meaning. Please, inline it